src/settings.ts

```diff
@@ -11,7 +13,7 @@ export const commonJSModuleSystemUsed = () => config().jsModuleSystem === 'commo
 
 const isExperimentOn = (experiment) => (config().experiments || []).includes(experiment);
 
-export const isHooksForFunctionalComponentsExperimentOn = () => isExperimentOn('hooksForFunctionalComponents');
+export const hooksSupported = () => ReactVersion[0] >= 16 && ReactVersion[1] >= 8;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 8](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)